compositor_common/src/error.rs

```diff
@@ -62,19 +62,12 @@ pub enum BuiltinSpecValidationError {
     FixedLayoutTopBottomOnlyOne,
     #[error("Fields \"left\" and \"right\" are mutually exclusive, you can only specify one in texture layout in \"fixed_position_layout\" transformation.")]
     FixedLayoutLeftRightOnlyOne,
-    #[error("Nodes that use transformation \"transform_to_resolution\" need to have exactly one input pad.")]
-    TransformToResolutionExactlyOneInput,
-    #[error(
-        "Transformation \"tiled_layout\" supports up to {} inputs.",
-        TILED_LAYOUT_MAX_INPUTS_COUNT
-    )]
-    TiledLayoutTooManyInputs,
-    #[error("Nodes that use transformation \"tiled_layout\" need to have at least one input pad.")]
-    TiledLayoutNoInputs,
-    #[error("Nodes that use transformation \"mirror_image\" need to have exactly one input pad.")]
-    MirrorImageExactlyOneInput,
-    #[error("Nodes that use transformation \"corner_radius\" need to have exactly one input pad.")]
-    CornerRadiusExactlyOneInput,
+    #[error("Nodes that use transformation \"{}\" expects {} input pads, but {} were provided", transformation_name, valid_input_pads_count.error_message_inputs_count(), defined_input_pads_count)]
```

Comment:
```suggestion
    #[error("Nodes that use transformation \"{}\" expect {} input pads, but {} were provided.", transformation_name, valid_input_pads_count.error_message_inputs_count(), defined_input_pads_count)]
```
I would prefer if you kept error messages for exactly on input separate from errors where expected number of inputs is constant but more than one.

Also, there is sth wrong with that wording "Nodes that use transformation \"{}\" expect" (Yes, I know that I'm probably an author)

Reply:
> I would prefer if you kept error messages for exactly on input separate from errors where expected number of inputs is constant but more than one.
 
Implementing a separate variant for 1 input without implementing a separate one for 0 inputs as well seems weird to me. I don't see benefits coming from treating one input differently than any other fixed input count value. We will have builtins that require other specific counts of inputs (e.g. fadings).
Do you have in mind any specific benefits related to separating it? I think that the error message would be similar / the same.

> Also, there is sth wrong with that wording "Nodes that use transformation "{}" expect" (Yes, I know that I'm probably an author)

Do you mean those parentheses are weird or sth else? I can remove them, no preference for that

Reply:
Ok, I'll try to fix it :)